test/e2e/apps/job.go

```diff
@@ -290,18 +290,8 @@ var _ = SIGDescribe("Job", func() {
 		framework.ExpectNoError(err, "failed to confirm that pods aren't created for job %s in namespace %s", job.Name, f.Namespace.Name)
```

Comment:
We can optimize this test a little:
After we waited for the job to have the suspend field, we can do a one time check that there are no pods (as we know that a sync already happened), instead of using `Consistently`, which is a busy wait.

Reply:
Pushed a change for that.